subprojects/tooling-api/src/testFixtures/groovy/org/gradle/integtests/tooling/fixture/ToolingApiSpecification.groovy

```diff
@@ -242,4 +242,12 @@ abstract class ToolingApiSpecification extends Specification {
     protected static GradleVersion getTargetVersion() {
         GradleVersion.version(targetDist.version.baseVersion.version)
     }
+
+    /**
+     * @return the current <b>base</b> version of Gradle being used to run this test
+     */
+    protected static GradleVersion getCurrentVersion() {
+        GradleVersion.current().baseVersion
+    }
+
```

Comment:
This will make it inconsistent with how we handle targetVersion.  I also think it will make it more difficult to read the code, not less.  Why do you think we should have these methods?